### ISSUE TITLE
connect: missing segment alert doesn't reappear

### ISSUE THREAD

**Issue by sshane** [commaai/connect#367]
https://connect.comma.ai/d5f0cbd9ceba7d0c/1687485479920/1687485661872

When clicking on a segment with no qcameras, `This video segment has not uploaded yet or has been deleted.` correctly shows. Then you click off that segment and it plays again, fine. Then when you click back to the segment with missing files, it just shows that it's loading indefinitely. It should show that same error again.

**Comment by adeebshihadeh** [comment]
Another case: https://connect.comma.ai/aa367395e351d0b2/1688655126537/1688657014434

**Comment by sshane** [comment]
those qcameras are missing, except the first

**Comment by sshane** [comment]
another instance https://connect.comma.ai/59658f79475526e4/1694401030869/1694401212084

https://github.com/commaai/connect/assets/25857203/e8238393-088b-481e-8186-4c400e208894

@incognitojam is this an easy fix?

**Comment by incognitojam** [comment]
No, I think it involves changes to Hls.js/react player since it isn't reporting the error fetching the same mpeg-ts segment again. I'll be looking at this after the prime stuff
